packages/arb-bridge-eth/contracts/bridge/interfaces/IBridge.sol

```diff
@@ -16,7 +16,8 @@
  * limitations under the License.
  */
 
-pragma solidity ^0.6.11;
+// solhint-disable-next-line compiler-version
+pragma solidity >=0.4.21 <0.9.0;
```

Comment:
won't work on `0.4.x` since the `calldata` location wasn't available in function params

as in 
```solidity
function executeCall(
        address destAddr,
        uint256 amount,
        bytes calldata data
    ) external returns (bool success, bytes memory returnData);``` 
    
    ```suggestion
pragma solidity >=0.5.0 <0.9.0;
```

Reply:
Actually it's only available since 0.6.9
https://github.com/ethereum/solidity/releases/tag/v0.6.9

Reply:
for all variables, but before that you had data location for specific variables (I think only variable length ones)
https://docs.soliditylang.org/en/v0.5.3/types.html#data-location-and-assignment-behaviour